examples/js/postprocessing/ManualMSAARenderPass.js

```diff
@@ -20,6 +20,10 @@ THREE.ManualMSAARenderPass = function ( scene, camera ) {
 	this.sampleLevel = 4; // specified as n, where the number of samples is 2^n, so sampleLevel = 4, is 2^4 samples, 16.
 	this.unbiased = true;
 
+	// as we need to clear the buffer in this pass, clearColor must be set to something, defaults to black.
+	this.clearColor = ( clearColor !== undefined ) ? clearColor : new THREE.Color( 0, 0, 0 );
```

Comment:
Done.  I'll have another PR that adds another set of different functional to the ManualMSAARenderPass after this PR is accepted.  Just trying not to have too much in a single PR.